dataland-frontend/tests/e2e/specs/low/pages/LandingPage.ts

```diff
@@ -3,7 +3,8 @@ describe("Login Section", () => {
     cy.visitAndCheckAppMount("/");
     cy.get("h1").should("contain.text", "CREATE A DATASET");
     cy.get('img[alt="Dataland logo"]').should("be.visible").should("have.attr", "src").should("include", "vision");
-    cy.get("button[name=get_started_button]").should("be.visible").should("contain.text", "Get Started");
+    cy.get("button[name=join_dataland_button]").should("be.visible").should("contain.text", "Join Dataland");
+    cy.get('i[alt="chevron_right"]').should("be.visible");
```

Comment:
Ich finde der Test sollte zusätzlich noch sicherstellen, dass man auf die Register Seite weitergeleitet wird, wenn man hier drauf klickt. Erst dann ist das feature für mich voll durch einen Test abgedeckt.

Reply:
Aber es gibt doch einen Test in AuthenticationButtons der die Knöpfe alle drückt und auch die jeweiligen Masken ausfüllt. Hier noch einen Test zu integrieren wäre doch doppelt, oder?

Reply:
Du hast recht. Passt.